<!DOCTYPE html>
<html lang="en">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link href="https://cdn.rawgit.com/dreampulse/computer-modern-web-font/master/fonts.css" rel="stylesheet">
<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Research | Tessa Han</title>
<meta name="generator" content="Jekyll v3.9.2" />
<meta property="og:title" content="Research" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Personal webpage of Tessa Han" />
<meta property="og:description" content="Personal webpage of Tessa Han" />
<link rel="canonical" href="http://localhost:4000/research" />
<meta property="og:url" content="http://localhost:4000/research" />
<meta property="og:site_name" content="Tessa Han" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Research" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","description":"Personal webpage of Tessa Han","headline":"Research","url":"http://localhost:4000/research"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="Tessa Han" /></head>
<body><header class="site-header" role="banner">

  <div class="wrapper"><a class="site-title" rel="author" href="/">Tessa Han</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"/>
            </svg>
          </span> 
        </label>

        <div class="trigger"><a class="page-link" href="/">About</a><a class="page-link" href="/research">Research</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <!-- ## Publications -->
<h2 style="font-weight:300">Publications</h2>

<ul>

<li>
 <p>
    Which Explanation Should I Choose? A Function Approximation Perspective to Characterizing Post Hoc Explanations <br />
    
       
        <b> Tessa Han</b>,    
        
      
       Suraj Srinivas, 
        
       and Hima Lakkaraju<br />
     <i>Neural Information Processing Systems (NeurIPS)</i>, 2022 <br />
      

<a class="collapsible"> synopsis</a>
<span class="content">
What are the basic properties of post hoc explanation methods and when is each method effective? Here, we 1) formalize the local function approximation framework, unifying eight diverse, popular explanation methods; 2) demonstrate that no method can perform optimally in every situation via the no free lunch theorem, calling for a principled approach to selecting among methods; and 3) set forth a guiding principle based on faithful function approximation to guide the selection among existing explanation methods and the design of novel ones.
</span>


  / <a href="https://arxiv.org/abs/2206.01254"> publication</a>
  




  / <a href="https://github.com/AI4LIFE-GROUP/lfa"> code</a>
  




<!-- uncomment to show workshop link -->
 <!-- / <a href="https://icml.cc/virtual/2022/workshop/13449"> workshop version </a> -->


<!-- vitessce paper -->


 <br />

  &gt;&gt; Best Paper Award, ICML Workshop on Interpretable Machine Learning for Healthcare (IMLH), 2022 <br />
  
  &gt;&gt; Oral Spotlight Presentation, ICML IMLH Workshop, 2022 <br />
  



<!-- disagreement paper -->






</p>
</li>

<li>
 <p>
    The Disagreement Problem in Explainable Machine Learning -- A Practitioner’s Perspective <br />
    
       Satyapriya Krishna*, 
        
      
       <b>Tessa Han</b>*, 
        
      
       Alex Gu, 
        
      
       Javin Pombra, 
        
      
       Shahin Jabbari, 
        
      
       Steven Wu, 
        
       and Hima Lakkaraju (*equal contribution)<br />
    
      <i>In submission for publication</i> <br />
      

<a class="collapsible"> synopsis</a>
<span class="content">
To what extent do explanations from post hoc explanation methods agree and how are disagreeing explanations resolved in practice? Here, we introduce and systematically study the “disagreement problem” in explainable machine learning. We find that existing explanation methods often generate disagreeing explanations and practitioners do not have a principled approach to selecting among disagreeing explanations, resorting to ad hoc heuristics such as personal preference.
</span>


  / <a href="https://arxiv.org/abs/2202.01602"> publication</a>
  








<!-- uncomment to show workshop link -->
 <!-- / <a href="https://chi-trait.github.io/#/schedule"> workshop version </a> -->


<!-- vitessce paper -->





<!-- disagreement paper -->
 <br />
&gt;&gt; Oral Spotlight Presentation, ICML Workshop on Interpretable Machine Learning for Healthcare (IMLH), 2022 <br />
&gt;&gt; Presentation, ACM CHI Workshop on Trust and Reliance in AI-Human Teams (TRAIT), 2022 <br />
&gt;&gt; Featured in Fortune (<a href="https://fortune.com/2022/02/08/learning-to-code-deepmind-alphacode/">Article 1</a> and <a href="https://fortune.com/2022/03/22/ai-explainable-radiology-medicine-crisis-eye-on-ai/">Article 2</a>) <br />
&gt;&gt; Featured in Analytics India Magazine (<a href="https://analyticsindiamag.com/explainable-ais-disagreement-problem/">Article</a>) <br />






</p>
</li>

<li>
 <p>
    Python and R packages for Interactive Visualization of Spatial Single-Cell Omics Data <br />
    
       Mark Keller, 
        
      
       Trevor Manz, 
        
      
       
        <b> Tessa Han</b>,    
        
      
       Ilan Gold, 
        
      
       Chuck McCallum, 
        
       and Nils Gehlenborg<br />
     <i>Conference on Intelligent Systems for Molecular Biology and European Conference on Computational Biology (ISMB/ECCB)</i>, 2021 <br />
      

<a class="collapsible"> synopsis</a>
<span class="content">
How do we effectively integrate and visualize high-dimensional, multimodal, spatial data in single-cell genomics? Here, to enable these visualizations, we develop Vitessce (Visual integration tool for exploration of spatial single cell experiments), a web-based, interactive visualization tool. We also develop associated Python, R/RStudio, and RShiny tools to enable using Vitessce within the Python and R bioinformatics ecosystems.
</span>


  / <a href="https://www.youtube.com/watch?v=xsb-zz8JrVU"> publication</a>
  









<!-- vitessce paper -->
 <br />
Tools: <a href="http://vitessce.io/"> Vitessce</a> + <a href="https://github.com/vitessce/vitessce"> code</a> / <a href="https://vitessce.github.io/vitessce-python/"> Python package</a> + <a href="https://github.com/vitessce/vitessce-python"> code</a> / <a href="https://vitessce.github.io/vitessceR/"> R package</a> + <a href="https://github.com/vitessce/vitessceR/"> code</a> / <a href="https://gehlenborglab.shinyapps.io/vitessce-shiny/"> R Shiny</a> + <a href="https://github.com/th789/vitessce-shiny"> code</a>





<!-- disagreement paper -->






</p>
</li>

<li>
 <p>
    Genome-Wide Somatic Alterations in Multiple Myeloma Reveal a Superior Outcome Group <br />
    
       Mehmet Kemal Samur, 
        
      
       Anil Aktas Samur, 
        
      
       Mariateresa Fulciniti, 
        
      
       Raphael Szalat, 
        
      
       
        <b> Tessa Han</b>,    
        
      
       Masood Shammas, 
        
      
       Paul Richardson, 
        
      
       Florence Magrangeas, 
        
      
       Stephane Minvielle, 
        
      
       Jill Corre, 
        
      
       Philippe Moreau, 
        
      
       Anjan Thakurta, 
        
      
       Kenneth Anderson, 
        
      
       Giovanni Parmigiani, 
        
      
       Hervé Avet-Loiseau, 
        
       and Nikhil Munshi<br />
     <i>Journal of Clinical Oncology</i>, 2020 <br />
      

<a class="collapsible"> synopsis</a>
<span class="content">
What is the genomic landscape of multiple myeloma and what are the genomic markers associated long-term survival? Here, we characterize the genomic landscape of multiple myeloma using deep whole-genome sequencing data. We find mutational signatures that reveal mechanisms of progression and variability in mutational load among known subgroups. In addition, we identify a new, genomically-defined subgroup (characterized by lower mutational load, age-related mutations, and the NRAS mutation) with prolonged overall and progression-free survival.
</span>


  / <a href="https://ascopubs.org/doi/full/10.1200/JCO.20.00461"> publication</a>
  









<!-- vitessce paper -->





<!-- disagreement paper -->






</p>
</li>

<li>
 <p>
    Dysregulated miRNAs after Uniform Treatment Predict Outcome of Newly-Diagnosed Multiple Myeloma <br />
    
       
        <b> Tessa Han</b>,    
        
      
       Stephane Minvielle, 
        
      
       Anil Aktas Samur, 
        
      
       Mariateresa Fulciniti, 
        
      
       Florence Magrangeas, 
        
      
       Paul Richardson, 
        
      
       Philippe Moreau, 
        
      
       Michel Attal, 
        
      
       Kenneth Anderson, 
        
      
       Hervé Avet-Loiseau, 
        
      
       Giovanni Parmigiani, 
        
      
       Mehmet Kemal Samur, 
        
       and Nikhil Munshi<br />
     <i>Blood (Supplemental Issue)</i>, 2019 <br />
      

<a class="collapsible"> synopsis</a>
<span class="content">
What is the role of miRNAs in the prognosis and progression of multiple myeloma? Here, we find that dysregulated miRNAs are predictive of disease prognosis in newly-diagnosed multiple myeloma patients and may contribute to disease progression by targeting key multiple myeloma genes (including MAF, BCL2, 14-3-3 and MYEOV).
</span>




  / <a href="https://ashpublications.org/blood/article/134/Supplement_1/4348/424591/Dysregulated-Mirnas-after-Uniform-Treatment"> publication (abstract)</a>
  







<!-- vitessce paper -->


 <br />

  &gt;&gt; Abstract Achievement Award, American Society of Hematology, 2019 <br />
  



<!-- disagreement paper -->






</p>
</li>

<li>
 <p>
    The Landscape of Genome Wide Somatic Alterations Identifies a Good-Risk Group in Newly-Diagnosed Multiple Myeloma <br />
    
       Mehmet Kemal Samur, 
        
      
       Anil Aktas Samur, 
        
      
       
        <b> Tessa Han</b>,    
        
      
       Marco Roncador, 
        
      
       Mariateresa Fulciniti, 
        
      
       Raphael Szalat, 
        
      
       Masood Shammas, 
        
      
       Jill Corre, 
        
      
       Paul Richardson, 
        
      
       Florence Magrangeas, 
        
      
       Stephane Minvielle, 
        
      
       Philippe Moreau, 
        
      
       Michel Attal, 
        
      
       Anjan Thakurta, 
        
      
       Kenneth Anderson, 
        
      
       Giovanni Parmigiani, 
        
      
       Hervé Avet-Loiseau, 
        
       and Nikhil Munshi<br />
     <i>Blood (Supplemental Issue)</i>, 2019 <br />
      

<a class="collapsible"> synopsis</a>
<span class="content">
At the genomic level, what distinguishes low-risk (long-survival) multiple myeloma patients from high-risk (short-survival) multiple myeloma patients? Here, we analyze whole-genome and whole-transcriptome sequencing data from multiple myeloma patients and identify a new low-risk subgroup in which chromosome and telomere maintenance pathways and genes associated with low bone disease are upregulated, processes associated with DNA damage are downregulated, and there are lower mutation rates for known oncogenes and tumor suppressors (e.g. TP53, CSMD1, and FAT3/FAT4).
</span>




  / <a href="https://ashpublications.org/blood/article/134/Supplement_1/3055/423505/The-Landscape-of-Genome-Wide-Somatic-Alterations"> publication (abstract)</a>
  







<!-- vitessce paper -->





<!-- disagreement paper -->






</p>
</li>

</ul>

<style>
p{
  padding-bottom: 10px;
}

.content {
  padding: 10px 0px;
  display: none;
  overflow: hidden;
}

</style>

<script>
  var coll = document.getElementsByClassName("collapsible");
  var i;

  for (i = 0; i < coll.length; i++) {
    coll[i].addEventListener("click", function() {
      this.classList.toggle("active");
      var content = this.nextElementSibling;
      if (content.style.display === "block") {
        content.style.display = "none";
      } else {
        content.style.display = "block";
      }
    });
  }
  </script>

<!-- old -->
<!-- - Which Explanation Should I Choose? A Function Approximation Perspective to Characterizing Post Hoc Explanations \
**Tessa Han**, Suraj Srinivas, and Himabindu Lakkaraju \
*Neural Information Processing Systems (NeurIPS)*, 2022 \
\>\> Oral Spotlight Presentation, ICML Workshop on Interpretable Machine Learning for Healthcare (IMLH), 2022 \
\>\> Best Paper Award, IMLH  -->

<!-- - The Disagreement Problem in Explainable Machine Learning: A Practitioner’s Perspective \
Satyapriya Krishna\*, **Tessa Han**\*, Alex Gu, Javin Pombra, Shahin Jabbari, Steven Wu, and Himabindu Lakkaraju (\*equal contribution) \
In submission to *Nature Machine Intelligence*, 2022 \
\>\> Oral Spotlight Presentation, ICML Workshop on Interpretable Machine Learning for Healthcare (IMLH), 2022 \
\>\> Presentation, ACM CHI Workshop on Trust and Reliance in AI-Human Teams (TRAIT), 2022 \
\>\> Featured in Fortune ([Article 1](https://fortune.com/2022/02/08/learning-to-code-deepmind-alphacode/) and [Article 2](https://fortune.com/2022/03/22/ai-explainable-radiology-medicine-crisis-eye-on-ai/)) \
\>\> Featured in Analytics India Magazine ([Article](https://analyticsindiamag.com/explainable-ais-disagreement-problem/)) -->

<!-- - Python and R packages for Interactive Visualization of Spatial Single-Cell Omics Data \
Mark Keller, Trevor Manz, **Tessa Han**, Ilan Gold, Chuck McCallum, and Nils Gehlenborg \
*Conference on Intelligent Systems for Molecular Biology and European Conference on Computational Biology (ISMB/ECCB)*, 2021 -->

<!-- - Genome-Wide Somatic Alterations in Multiple Myeloma Reveal a Superior Outcome Group \
Mehmet Kemal Samur, Anil Aktas Samur, Mariateresa Fulciniti, Raphael Szalat, **Tessa Han**, Masood Shammas, Paul Richardson, Florence Magrangeas, Stephane Minvielle, Jill Corre, Philippe Moreau, Anjan Thakurta, Kenneth C. Anderson, Giovanni Parmigiani, Hervé Avet-Loiseau, and Nikhil C. Munshi \
*Journal of Clinical Oncology*, 2020 -->

<!-- - Dysregulated miRNAs after Uniform Treatment Predict Outcome of Newly-Diagnosed Multiple Myeloma \
**Tessa Han**, Stephane Minvielle, Anil Aktas Samur, Mariateresa Fulciniti, Florence Magrangeas, Paul G. Richardson, Philippe Moreau, Michel Attal, Kenneth C. Anderson, Hervé Avet-Loiseau, Giovanni Parmigiani, Mehmet Kemal Samur, and Nikhil C. Munshi \
*Blood* (Supplemental Issue), 2019 \
\>\> Abstract Achievement Award, American Society of Hematology -->

<!-- - The Landscape of Genome Wide Somatic Alterations Identifies a Good-Risk Group in Newly-Diagnosed Multiple Myeloma \
Mehmet Kemal Samur, Anil Aktas Samur, **Tessa Han**, Marco Roncador, Mariateresa Fulciniti, Raphael Szalat, Masood A. Shammas, Jill Corre, Paul G. Richardson, Florence Magrangeas, Stephane Minvielle, Philippe Moreau, Michel Attal, Anjan Thakurta, Kenneth C. Anderson, Giovanni Parmigiani, Herve Avet-Loiseau, and Nikhil C. Munshi \
*Blood* (Supplemental Issue), 2019 -->

<!-- ## Thesis -->
<h2 style="font-weight:300">Thesis</h2>

<!-- 
<ul>

<li>
 <p>
    Applying Mediation Analysis to Predict Feed Forward Loops as Gene Regulatory Networks in Multiple Myeloma -- A Novel Statistical Approach <br>
    Honors thesis for a joint A.B. degree in Statistics and Molecular and Cellular Biology, 2020 <br>
    Advisor: Prof. Giovanni Parmigiani (Harvard University) <br>


<a class="collapsible"> synopsis</a>
<span class="content">
How do we predict feed forward loops (FFLs), a type of gene regulatory network, using gene and miRNA expression profiles? What do predicted FFLs in multiple myeloma reveal about the biology of the disease? This thesis posits a novel computational method to predict FFLs using mediation analysis. Applying the method to analyzing a multiple myeloma dataset predicts numerous FFLs that involve known oncogenes and tumor suppressors. Upon experimental validation, these findings may lead to the discovery of the first FFLs in multiple myeloma, elucidating the biology that underlies the disease and revealing new disease-level and subgroup-specific treatment strategies for multiple myeloma patients.
</span>








 <br>

  >> Hoopes Prize for “outstanding scholarly work or research”, Harvard University, 2020 <br>
  





</p>
</li>

</ul>

<style>
p{
  padding-bottom: 10px;
}

.content {
  padding: 10px 0px;
  display: none;
  overflow: hidden;
}

</style>

<script>
  var coll = document.getElementsByClassName("collapsible");
  var i;

  for (i = 0; i < coll.length; i++) {
    coll[i].addEventListener("click", function() {
      this.classList.toggle("active");
      var content = this.nextElementSibling;
      if (content.style.display === "block") {
        content.style.display = "none";
      } else {
        content.style.display = "block";
      }
    });
  }
  </script> -->

<ul>
  <li>Applying Mediation Analysis to Predict Feed Forward Loops as Gene Regulatory Networks in Multiple Myeloma – A Novel Statistical Approach <br />
Honors thesis for a joint A.B. degree in Statistics and Molecular and Cellular Biology <br />
Supervisor: Prof. Giovanni Parmigiani (Harvard University) <br />
&gt;&gt; Hoopes Prize for “outstanding scholarly work or research”, Harvard University, 2020</li>
</ul>

      </div>
    </main>

  </body>

  <style>
    body {
        font-size: 18px;
        font-weight: 300;
    }

    b {
    font-weight: 400;
    }
    </style>

</html>
