<!DOCTYPE html>
<html lang="en">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link href="https://cdn.rawgit.com/dreampulse/computer-modern-web-font/master/fonts.css" rel="stylesheet">
<link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400&display=swap" rel="stylesheet"><head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <meta name="viewport" content="width=device-width, initial-scale=1"><!-- Begin Jekyll SEO tag v2.8.0 -->
<title>Research | Tessa Han</title>
<meta name="generator" content="Jekyll v3.9.2" />
<meta property="og:title" content="Research" />
<meta property="og:locale" content="en_US" />
<meta name="description" content="Personal webpage of Tessa Han" />
<meta property="og:description" content="Personal webpage of Tessa Han" />
<link rel="canonical" href="http://localhost:4000/research" />
<meta property="og:url" content="http://localhost:4000/research" />
<meta property="og:site_name" content="Tessa Han" />
<meta property="og:type" content="website" />
<meta name="twitter:card" content="summary" />
<meta property="twitter:title" content="Research" />
<script type="application/ld+json">
{"@context":"https://schema.org","@type":"WebPage","description":"Personal webpage of Tessa Han","headline":"Research","url":"http://localhost:4000/research"}</script>
<!-- End Jekyll SEO tag -->
<link rel="stylesheet" href="/assets/main.css"><link type="application/atom+xml" rel="alternate" href="http://localhost:4000/feed.xml" title="Tessa Han" /></head>
<body><header class="site-header" role="banner">

  <div class="wrapper"><a class="site-title" rel="author" href="/">Tessa Han</a><nav class="site-nav">
        <input type="checkbox" id="nav-trigger" class="nav-trigger" />
        <label for="nav-trigger">
          <span class="menu-icon">
            <svg viewBox="0 0 18 15" width="18px" height="15px">
              <path d="M18,1.484c0,0.82-0.665,1.484-1.484,1.484H1.484C0.665,2.969,0,2.304,0,1.484l0,0C0,0.665,0.665,0,1.484,0 h15.032C17.335,0,18,0.665,18,1.484L18,1.484z M18,7.516C18,8.335,17.335,9,16.516,9H1.484C0.665,9,0,8.335,0,7.516l0,0 c0-0.82,0.665-1.484,1.484-1.484h15.032C17.335,6.031,18,6.696,18,7.516L18,7.516z M18,13.516C18,14.335,17.335,15,16.516,15H1.484 C0.665,15,0,14.335,0,13.516l0,0c0-0.82,0.665-1.483,1.484-1.483h15.032C17.335,12.031,18,12.695,18,13.516L18,13.516z"/>
            </svg>
          </span> 
        </label>

        <div class="trigger"><a class="page-link" href="/">About</a><a class="page-link" href="/research">Research</a></div>
      </nav></div>
</header>
<main class="page-content" aria-label="Content">
      <div class="wrapper">
        <!-- ## Publications -->
<h2 style="font-weight:400">Publications</h2>

<p>For a more up-to-date list, please see <a href="https://scholar.google.com/citations?user=exlT558AAAAJ&amp;hl=en">my Google Scholar profile</a>.</p>

<!-- Navigation Buttons -->
<div class="tabs">
  <ul class="tab-list">
    <li class="tab-item"><a href="#selected-publications-header">selected publications</a></li>
    <li class="tab-item"><a href="#all-publications-and-preprints-header">all publications and preprints</a></li>
  </ul>
</div>

<!-- Content Sections -->
<div id="selected-publications">
  <h2 id="selected-publications-header" style="font-weight:300">Selected publications</h2>
  <ul>
    
    
    
    
    <li>
      <p>
          MedSafetyBench – Evaluating and Improving the Medical Safety of Large Language Models <br />
          
             
              <b> Tessa Han</b>,    
              
            
             Aounon Kumar, 
              
            
             Chirag Agarwal, 
              
             and Hima Lakkaraju<br />
           <i>Conference on Neural Information Processing Systems (NeurIPS), Datasets and Benchmarks</i>, 2024 <br />
            
        
          <!-- summary + links for paper -->
          <a class="collapsible"> summary</a>
          <span class="content">
          How do we define, evaluate, and improve the medical safety of large language models (LLMs)? In this paper, we define the notion of medical safety for LLMs, develop a novel benchmark dataset (MedSafetyBench) to assess it, and evaluate and improve the medical safety of medical LLMs. We find that 1) current medical LLMs do not meet standards of general or medical safety and that 2) fine-tuning these medical LLMs using our benchmark dataset significantly improves their safety.
          </span>
          
          
            / <a href="https://arxiv.org/abs/2403.03744" target="_blank"> paper</a>
            
          
          
            / <a href="https://github.com/AI4LIFE-GROUP/med-safety-bench" target="_blank"> code</a>
            
          
            / <a href="https://github.com/AI4LIFE-GROUP/med-safety-bench/tree/main/datasets" target="_blank"> dataset</a>
            
          
          
          <!-- notes for paper: awards, media -->
          

        
        
      </p>
    </li>
    
    
    
    <li>
      <p>
          Characterizing Data Point Vulnerability via Average-Case Robustness <br />
          
             
              <b> Tessa Han*</b>,    
              
            
             Suraj Srinivas*, 
              
             and Hima Lakkaraju<br />
           <i>Conference on Uncertainty in Artificial Intelligence (UAI)</i>, 2024 <br />
            
        
          <!-- summary + links for paper -->
          <a class="collapsible"> summary</a>
          <span class="content">
          How do we efficiently characterize a data point's vulnerability using average-case robustness? In this paper, we 1) derive efficient analytical estimators, 2) validate their accuracy and efficiency through theoretical and empirical analyses, and 3) demonstrate their usefulness for various tasks involving average-case robustness (e.g., measuring class-level robustness bias and identifying dataset samples that are vulnerable to noise perturbation).
          </span>
          
          
            / <a href="https://arxiv.org/abs/2307.13885" target="_blank"> paper</a>
            
          
          
            / <a href="https://github.com/AI4LIFE-GROUP/average-case-robustness" target="_blank"> code</a>
            
          
          
          
          <!-- notes for paper: awards, media -->
          

        
        
      </p>
    </li>
    
    
    
    <li>
      <p>
          Which Explanation Should I Choose? A Function Approximation Perspective to Characterizing Post Hoc Explanations <br />
          
             
              <b> Tessa Han</b>,    
              
            
             Suraj Srinivas, 
              
             and Hima Lakkaraju<br />
           <i>Conference on Neural Information Processing Systems (NeurIPS)</i>, 2022 <br />
            
        
          <!-- summary + links for paper -->
          <a class="collapsible"> summary</a>
          <span class="content">
          What are the basic properties of post hoc explanation methods and when is each method effective? In this paper, we 1) formalize the local function approximation framework, unifying eight diverse, popular explanation methods; 2) demonstrate that no method can perform optimally in every situation via the no free lunch theorem, calling for a principled approach to selecting among methods; and 3) set forth a guiding principle based on faithful function approximation to guide the selection among existing explanation methods and the design of novel ones.
          </span>
          
          
            / <a href="https://arxiv.org/abs/2206.01254" target="_blank"> paper</a>
            
          
          
            / <a href="https://github.com/AI4LIFE-GROUP/lfa" target="_blank"> code</a>
            
          
          
          
          <!-- notes for paper: awards, media -->
           <br />
          
            &gt;&gt; <span style="background-color:#d2efdb">Best Paper Award</span>, ICML Workshop on Interpretable ML for Healthcare (IMLH), 2022 <br />
            
          

        
        
      </p>
    </li>
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
    
  </ul>
</div>

<!--- all publications and preprints -->
<div id="all-publications-and-preprints">
  <h2 id="all-publications-and-preprints-header" style="font-weight:300; margin-top: 30px;">All publications and preprints</h2>
  <ul>
    
    <li>
      <p>
        (preprint) Building Bridges, Not Walls – Advancing Interpretability by Unifying Feature, Data, and Model Component Attribution <br />
        
           Shichang Zhang, 
            
          
           
            <b> Tessa Han*</b>,    
            
          
           Usha Bhalla*, 
            
           and Hima Lakkaraju<br />
        
          <!-- <i>Preprint</i> <br> -->
          <i> </i>
          
    
        <!-- summary + links for paper -->
        <a class="collapsible"> summary</a>
        <span class="content">
        In intepretability research, the areas of feature, data, and component attribution tend to be studied and applied rather independently, resulting in a fragmented research landscape. In this position paper, we argue that 1) these three areas share fundamental similarities, employing common approaches and differing mainly in perspective rather than core techniques, and that 2) this more unified view can advance not only interpretability research, by stimulating cross-domain knowledge transfer, but also broader AI research, by improving model editing, steering, and regulation.
        </span>
        
        
          / <a href="https://arxiv.org/pdf/2501.18887" target="_blank"> paper</a>
        
        
        
        
        
        
        <!-- vitessce paper -->
        
        
        <!-- notes for paper: awards, media -->
        
        
      </p>
    </li>
    
    <li>
      <p>
        MedSafetyBench – Evaluating and Improving the Medical Safety of Large Language Models <br />
        
           
            <b> Tessa Han</b>,    
            
          
           Aounon Kumar, 
            
          
           Chirag Agarwal, 
            
           and Hima Lakkaraju<br />
         <i>Conference on Neural Information Processing Systems (NeurIPS), Datasets and Benchmarks</i>, 2024 <br />
          
    
        <!-- summary + links for paper -->
        <a class="collapsible"> summary</a>
        <span class="content">
        How do we define, evaluate, and improve the medical safety of large language models (LLMs)? In this paper, we define the notion of medical safety for LLMs, develop a novel benchmark dataset (MedSafetyBench) to assess it, and evaluate and improve the medical safety of medical LLMs. We find that 1) current medical LLMs do not meet standards of general or medical safety and that 2) fine-tuning these medical LLMs using our benchmark dataset significantly improves their safety.
        </span>
        
        
          / <a href="https://arxiv.org/abs/2403.03744" target="_blank"> paper</a>
        
        
        
          / <a href="https://github.com/AI4LIFE-GROUP/med-safety-bench" target="_blank"> code</a>
        
        
          / <a href="https://github.com/AI4LIFE-GROUP/med-safety-bench/tree/main/datasets" target="_blank"> dataset</a>
        
        
        
        <!-- vitessce paper -->
        
        
        <!-- notes for paper: awards, media -->
        
        
      </p>
    </li>
    
    <li>
      <p>
        Characterizing Data Point Vulnerability via Average-Case Robustness <br />
        
           
            <b> Tessa Han*</b>,    
            
          
           Suraj Srinivas*, 
            
           and Hima Lakkaraju<br />
         <i>Conference on Uncertainty in Artificial Intelligence (UAI)</i>, 2024 <br />
          
    
        <!-- summary + links for paper -->
        <a class="collapsible"> summary</a>
        <span class="content">
        How do we efficiently characterize a data point's vulnerability using average-case robustness? In this paper, we 1) derive efficient analytical estimators, 2) validate their accuracy and efficiency through theoretical and empirical analyses, and 3) demonstrate their usefulness for various tasks involving average-case robustness (e.g., measuring class-level robustness bias and identifying dataset samples that are vulnerable to noise perturbation).
        </span>
        
        
          / <a href="https://arxiv.org/abs/2307.13885" target="_blank"> paper</a>
        
        
        
          / <a href="https://github.com/AI4LIFE-GROUP/average-case-robustness" target="_blank"> code</a>
        
        
        
        
        <!-- vitessce paper -->
        
        
        <!-- notes for paper: awards, media -->
        
        
      </p>
    </li>
    
    <li>
      <p>
        Which Explanation Should I Choose? A Function Approximation Perspective to Characterizing Post Hoc Explanations <br />
        
           
            <b> Tessa Han</b>,    
            
          
           Suraj Srinivas, 
            
           and Hima Lakkaraju<br />
         <i>Conference on Neural Information Processing Systems (NeurIPS)</i>, 2022 <br />
          
    
        <!-- summary + links for paper -->
        <a class="collapsible"> summary</a>
        <span class="content">
        What are the basic properties of post hoc explanation methods and when is each method effective? In this paper, we 1) formalize the local function approximation framework, unifying eight diverse, popular explanation methods; 2) demonstrate that no method can perform optimally in every situation via the no free lunch theorem, calling for a principled approach to selecting among methods; and 3) set forth a guiding principle based on faithful function approximation to guide the selection among existing explanation methods and the design of novel ones.
        </span>
        
        
          / <a href="https://arxiv.org/abs/2206.01254" target="_blank"> paper</a>
        
        
        
          / <a href="https://github.com/AI4LIFE-GROUP/lfa" target="_blank"> code</a>
        
        
        
        
        <!-- vitessce paper -->
        
        
        <!-- notes for paper: awards, media -->
         <br />
        
          &gt;&gt; <span style="background-color:#d2efdb">Best Paper Award</span>, ICML Workshop on Interpretable ML for Healthcare (IMLH), 2022 <br />
          
        
        
      </p>
    </li>
    
    <li>
      <p>
        The Disagreement Problem in Explainable Machine Learning – A Practitioner’s Perspective <br />
        
           Satyapriya Krishna*, 
            
          
           
            <b> Tessa Han*</b>,    
            
          
           Alex Gu, 
            
          
           Javin Pombra, 
            
          
           Shahin Jabbari, 
            
          
           Steven Wu, 
            
           and Hima Lakkaraju<br />
         <i>Transactions on Machine Learning Research (TMLR)</i>, 2024 <br />
          
    
        <!-- summary + links for paper -->
        <a class="collapsible"> summary</a>
        <span class="content">
        To what extent do explanations from post hoc explanation methods agree and how are disagreeing explanations resolved in practice? In this paper, we introduce and systematically study the “disagreement problem” in explainable machine learning. We find that 1) existing explanation methods often generate disagreeing explanations and 2) practitioners do not have a principled approach to selecting among disagreeing explanations, resorting to ad hoc heuristics such as personal preference.
        </span>
        
        
          / <a href="https://arxiv.org/abs/2202.01602" target="_blank"> paper</a>
        
        
        
        
        
        
        <!-- vitessce paper -->
        
        
        <!-- notes for paper: awards, media -->
         <br />
        
          &gt;&gt; <span style="background-color:#d2efdb">Media coverage</span> - Featured in Fortune (<a href="https://fortune.com/2022/02/08/learning-to-code-deepmind-alphacode/" target="_blank">Article 1</a> and <a href="https://fortune.com/2022/03/22/ai-explainable-radiology-medicine-crisis-eye-on-ai/" target="_blank">Article 2</a>) and Analytics India Magazine (<a href="https://analyticsindiamag.com/explainable-ais-disagreement-problem/" target="_blank">Article</a>) <br />
          
        
        
      </p>
    </li>
    
    <li>
      <p>
        Is Ignorance Bliss? Investigating How Faithfulness and Alignment of Post Hoc Explanations Affect Model Trust in Laypeople and Domain Experts <br />
        
           
            <b> Tessa Han*</b>,    
            
          
           Yasha Ektefaie*, 
            
          
           Maha Farhat, 
            
          
           Marinka Zitnik, 
            
           and Hima Lakkaraju<br />
         <i>Machine Learning for Health Symposium (ML4H)</i>, 2023 <br />
          
    
        <!-- summary + links for paper -->
        <a class="collapsible"> summary</a>
        <span class="content">
        What factors affect laypeople and domain experts' trust in post hoc explanations? In this paper, we find that while laypeople base their trust on explanation faithfulness, domain experts (incorrectly) base theirs on explanation alignment, suggesting a cognitive bias among domain experts.
        </span>
        
        
          / <a href="https://arxiv.org/abs/2312.05690" target="_blank"> paper</a>
        
        
        
        
        
        
        <!-- vitessce paper -->
        
        
        <!-- notes for paper: awards, media -->
        
        
      </p>
    </li>
    
    <li>
      <p>
        Hevelius Report – Visualizing Web-Based Motility Test Data for Clinical Decision and Learning Support <br />
        
           Hongjin Lin, 
            
          
           
            <b> Tessa Han</b>,    
            
          
           Krzysztof Gajos, 
            
           and Anoopum Gupta<br />
         <i>ACM SIGACCESS Conference on Computers and Accessibility (ASSETS)</i>, 2024 <br />
          
    
        <!-- summary + links for paper -->
        <a class="collapsible"> summary</a>
        <span class="content">
        How do we measure, visualize, and interpret the mobility of patients with Parkinson's or ataxia over time? In this work, we develop Hevelius Report, a visualization tool to track the mobility of these patients, and demonstrate its utility.
        </span>
        
        
          / <a href="https://arxiv.org/abs/2409.06088" target="_blank"> paper</a>
        
        
        
        
        
        
        <!-- vitessce paper -->
        
        
        <!-- notes for paper: awards, media -->
        
        
      </p>
    </li>
    
    <li>
      <p>
        Python and R packages for Interactive Visualization of Spatial Single-Cell Omics Data <br />
        
           Mark Keller, 
            
          
           Trevor Manz, 
            
          
           
            <b> Tessa Han</b>,    
            
          
           Ilan Gold, 
            
          
           Chuck McCallum, 
            
           and Nils Gehlenborg<br />
         <i>Conference on Intelligent Systems for Molecular Biology and European Conference on Computational Biology (ISMB/ECCB)</i>, 2021 <br />
          
    
        <!-- summary + links for paper -->
        <a class="collapsible"> summary</a>
        <span class="content">
        How do we effectively integrate and visualize high-dimensional, multimodal, spatial data in single-cell genomics? In this paper, to enable these visualizations, we develop Vitessce (Visual integration tool for exploration of spatial single cell experiments), a web-based, interactive visualization tool. We also develop associated Python, R/RStudio, and RShiny tools to enable using Vitessce within the Python and R bioinformatics ecosystems.
        </span>
        
        
          / <a href="https://www.youtube.com/watch?v=xsb-zz8JrVU" target="_blank"> paper</a>
        
        
        
        
        
        
        <!-- vitessce paper -->
         <br />
        Tools: <a href="http://vitessce.io/"> Vitessce</a> + <a href="https://github.com/vitessce/vitessce"> code</a> / <a href="https://vitessce.github.io/vitessce-python/"> Python package</a> + <a href="https://github.com/vitessce/vitessce-python"> code</a> / <a href="https://vitessce.github.io/vitessceR/"> R package</a> + <a href="https://github.com/vitessce/vitessceR/"> code</a> / <a href="https://gehlenborglab.shinyapps.io/vitessce-shiny/"> R Shiny</a> + <a href="https://github.com/th789/vitessce-shiny"> code</a>
        
        
        <!-- notes for paper: awards, media -->
        
        
      </p>
    </li>
    
    <li>
      <p>
        Genome-Wide Somatic Alterations in Multiple Myeloma Reveal a Superior Outcome Group <br />
        
           Mehmet Kemal Samur, 
            
          
           Anil Aktas Samur, 
            
          
           Mariateresa Fulciniti, 
            
          
           Raphael Szalat, 
            
          
           
            <b> Tessa Han</b>,    
            
          
           Masood Shammas, 
            
          
           Paul Richardson, 
            
          
           Florence Magrangeas, 
            
          
           Stephane Minvielle, 
            
          
           Jill Corre, 
            
          
           Philippe Moreau, 
            
          
           Anjan Thakurta, 
            
          
           Kenneth Anderson, 
            
          
           Giovanni Parmigiani, 
            
          
           Hervé Avet-Loiseau, 
            
           and Nikhil Munshi<br />
         <i>Journal of Clinical Oncology</i>, 2020 <br />
          
    
        <!-- summary + links for paper -->
        <a class="collapsible"> summary</a>
        <span class="content">
        What is the genomic landscape of multiple myeloma and what are the genomic markers associated long-term survival? In this paper, we characterize the genomic landscape of multiple myeloma using deep whole-genome sequencing data. We find mutational signatures that reveal mechanisms of progression and variability in mutational load among known subgroups. In addition, we identify a new, genomically-defined subgroup (characterized by lower mutational load, age-related mutations, and the NRAS mutation) with prolonged overall and progression-free survival.
        </span>
        
        
          / <a href="https://ascopubs.org/doi/full/10.1200/JCO.20.00461" target="_blank"> paper</a>
        
        
        
        
        
        
        <!-- vitessce paper -->
        
        
        <!-- notes for paper: awards, media -->
        
        
      </p>
    </li>
    
    <li>
      <p>
        Dysregulated miRNAs after Uniform Treatment Predict Outcome of Newly-Diagnosed Multiple Myeloma <br />
        
           
            <b> Tessa Han</b>,    
            
          
           Stephane Minvielle, 
            
          
           Anil Aktas Samur, 
            
          
           Mariateresa Fulciniti, 
            
          
           Florence Magrangeas, 
            
          
           Paul Richardson, 
            
          
           Philippe Moreau, 
            
          
           Michel Attal, 
            
          
           Kenneth Anderson, 
            
          
           Hervé Avet-Loiseau, 
            
          
           Giovanni Parmigiani, 
            
          
           Mehmet Kemal Samur, 
            
           and Nikhil Munshi<br />
         <i>Blood (Supplemental Issue)</i>, 2019 <br />
          
    
        <!-- summary + links for paper -->
        <a class="collapsible"> summary</a>
        <span class="content">
        What is the role of miRNAs in the prognosis and progression of multiple myeloma? Here, we find that dysregulated miRNAs are predictive of disease prognosis in newly-diagnosed multiple myeloma patients and may contribute to disease progression by targeting key multiple myeloma genes (including MAF, BCL2, 14-3-3 and MYEOV).
        </span>
        
        
        
          / <a href="https://ashpublications.org/blood/article/134/Supplement_1/4348/424591/Dysregulated-Mirnas-after-Uniform-Treatment" target="_blank"> paper (abstract)</a>
        
        
        
        
        
        <!-- vitessce paper -->
        
        
        <!-- notes for paper: awards, media -->
         <br />
        
          &gt;&gt; <span style="background-color:#d2efdb">Abstract Achievement Award</span>, American Society of Hematology, 2019 <br />
          
        
        
      </p>
    </li>
    
    <li>
      <p>
        The Landscape of Genome-Wide Somatic Alterations Identifies a Good-Risk Group in Newly-Diagnosed Multiple Myeloma <br />
        
           Mehmet Kemal Samur, 
            
          
           Anil Aktas Samur, 
            
          
           
            <b> Tessa Han</b>,    
            
          
           Marco Roncador, 
            
          
           Mariateresa Fulciniti, 
            
          
           Raphael Szalat, 
            
          
           Masood Shammas, 
            
          
           Jill Corre, 
            
          
           Paul Richardson, 
            
          
           Florence Magrangeas, 
            
          
           Stephane Minvielle, 
            
          
           Philippe Moreau, 
            
          
           Michel Attal, 
            
          
           Anjan Thakurta, 
            
          
           Kenneth Anderson, 
            
          
           Giovanni Parmigiani, 
            
          
           Hervé Avet-Loiseau, 
            
           and Nikhil Munshi<br />
         <i>Blood (Supplemental Issue)</i>, 2019 <br />
          
    
        <!-- summary + links for paper -->
        <a class="collapsible"> summary</a>
        <span class="content">
        At the genomic level, what distinguishes low-risk (long-survival) multiple myeloma patients from high-risk (short-survival) multiple myeloma patients? Here, we analyze whole-genome and whole-transcriptome sequencing data from multiple myeloma patients and identify a new low-risk subgroup in which chromosome and telomere maintenance pathways and genes associated with low bone disease are upregulated, processes associated with DNA damage are downregulated, and there are lower mutation rates for known oncogenes and tumor suppressors (e.g. TP53, CSMD1, and FAT3/FAT4).
        </span>
        
        
        
          / <a href="https://ashpublications.org/blood/article/134/Supplement_1/3055/423505/The-Landscape-of-Genome-Wide-Somatic-Alterations" target="_blank"> paper (abstract)</a>
        
        
        
        
        
        <!-- vitessce paper -->
        
        
        <!-- notes for paper: awards, media -->
        
        
      </p>
    </li>
    
  </ul>
</div>

<!-- CSS for Horizontal Tabs -->
<style>
  .tabs {
    margin-bottom: 20px; /* Adds space below the tabs */
  }

  .tab-list {
    list-style: none;
    padding: 0;
    margin: 0;
    display: flex;
    justify-content: flex-start; /* Align tabs to the left */
    gap: 20px; /* Add spacing between tabs */
  }

  .tab-item {
    display: inline;
  }

  .tab-item a {
    text-decoration: none;
    color: #007BFF; /* Link color */
    font-weight: bold;
    padding: 5px 10px;
    border-radius: 5px;
    transition: background-color 0.3s ease;
  }

  .tab-item a:hover,
  .tab-item a.active {
    background-color: #f0f0f0; /* Gray background for hover and active */
  }
</style>

<!--- make links the same color, even if previously visited-->
<style>
  a {
    color: #007BFF; /* Set a consistent blue color for all links */
    text-decoration: none; /* Optional: Remove underline */
  }

  a:visited {
    color: #007BFF; /* Ensure visited links have the same color */
  }

  a:hover {
    color: #0056b3; /* Optional: Slightly darker blue on hover */
    text-decoration: underline; /* Optional: Add underline on hover */
  }

  a:active {
    color: #0056b3; /* Optional: Same as hover for active state */
  }
</style>

<style>
p{
  padding-bottom: 10px;
}

.content {
  padding: 10px 0px;
  display: none;
  overflow: hidden;
}

</style>

<script>
  var coll = document.getElementsByClassName("collapsible");
  var i;

  for (i = 0; i < coll.length; i++) {
    coll[i].addEventListener("click", function() {
      this.classList.toggle("active");
      var content = this.nextElementSibling;
      if (content.style.display === "block") {
        content.style.display = "none";
      } else {
        content.style.display = "block";
      }
    });
  }
  </script>


      </div>
    </main>

  </body>

  <style>
    body {
        font-size: 18px;
        font-weight: 300;
    }

    b {
    font-weight: 400;
    }
    </style>

</html>
